samples/cpp/tutorial_code/ImgProc/Threshold_inRange.cpp

```diff
@@ -60,17 +60,17 @@ int main(int argc, char* argv[])
     //! [cap]
 
     //! [window]
-    namedWindow(window_capture_name);
-    namedWindow(window_detection_name);
+    namedWindow(window_capture_name,WINDOW_NORMAL);
+    namedWindow(window_detection_name,WINDOW_NORMAL);
```

Comment:
Probably it is a highgui module bug.
No need to change this sample (we are not going to change all users code)